src/accessors/ILogMessage.ts

```diff
@@ -0,0 +1,9 @@
+import { ILogMessageSeverity } from './LogMessageSeverity';
+/**
+ * Message which will be passed to a UI (either in a log or in the application's UI)
+ */
+export interface ILogMessage {
```

Comment:
@mrsimpson Rocket.Chat right now has two was of notifying a user something happened. If it is in the context of a room, we have Notifications which are temporary messages on the client side only and are visible to only the sent user. These notifications go away if you refresh the page. The other way is via the `Toastr`.

So, we could abstract away these two into one interface. I like that idea, this way it's not specific to one or the other and can be changed behind the scenes in the future but Rocketlets stay the same.